src/features/flows/thingy91x/Evaluate.tsx

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+import React, { useCallback, useEffect, useState } from 'react';
+import {
+    Button,
+    logger,
+    openUrl,
+    Spinner,
+    telemetry,
+} from '@nordicsemiconductor/pc-nrfconnect-shared';
+import describeError from '@nordicsemiconductor/pc-nrfconnect-shared/src/logging/describeError';
+
+import { useAppDispatch, useAppSelector } from '../../../app/store';
+import { Back } from '../../../common/Back';
+import Copy from '../../../common/Copy';
+import Main from '../../../common/Main';
+import { Next } from '../../../common/Next';
+import { IssueBox } from '../../../common/NoticeBox';
+import runVerification from '../../../common/sendATCommands';
+import { getSelectedDeviceUnsafely } from '../../device/deviceSlice';
+import {
+    getAttestationToken,
+    getFailed,
+    setAttestationToken,
+    setFailed,
+} from './thingy91xSlice';
+
+const nRFCloudLink =
+    'https://docs.nordicsemi.com/bundle/nrf-cloud/page/GettingStarted.html';
+
+export default () => {
+    const dispatch = useAppDispatch();
+    const device = useAppSelector(getSelectedDeviceUnsafely);
+    const token = useAppSelector(getAttestationToken);
+    const failed = useAppSelector(getFailed);
+    const [gettingToken, setGettingToken] = useState(false);
+
+    const getToken = useCallback(() => {
+        setGettingToken(true);
+        dispatch(setFailed(false));
+
+        runVerification(
+            [
+                {
+                    command: 'AT%ATTESTTOKEN',
+                    responseRegex: '%ATTESTTOKEN: "(.*)"',
+                },
+            ],
+            device.serialPorts?.[0].comName || '',
+            'SHELL'
+        )
+            .then(res => {
+                dispatch(setAttestationToken(res[0]));
+            })
+            .catch(e => {
+                logger.error(describeError(e));
+                dispatch(setFailed(true));
+            })
+            .finally(() => setGettingToken(false));
+    }, [device, dispatch]);
+
+    useEffect(() => {
+        if (!token && !failed && !gettingToken) {
+            getToken();
+        }
+    }, [token, failed, gettingToken, getToken]);
+
+    return (
+        <Main>
+            <Main.Content
+                heading="Evaluate nRF Cloud multi-service sample"
+                subHeading="Onboarding, data collection, FOTA updates, location, logging, alerts, and more."
+            >
+                <div className="tw-flex tw-flex-col tw-gap-4">
+                    <div>
+                        <b>Claim your device</b>
+                        <p>
+                            To evaluate all features you need to connect your DK
```

Comment:
Thingy devices are not DKs and should not be referred to as such. They are Prototyping Platforms, so this line should read `device` (or use the name of the device).